### DISCUSSION TITLE
Changing aeBinaryStrategy

### DISCUSSION THREAD

**Discussion by tolgaburcak** [inlife/nexrender#978] [General]
I have After Effects 2020,2023,2024 installed on my machine. I want to use 2024 but nexrender finds 2020 automatically. How can I set that?

 - aeBinaryStrategy: auto
 - binary: C:\Program Files\Adobe\Adobe After Effects 2022\Support Files\aerender.exe

**Comment by inlife** [comment]
Both CLI and Worker have `--binary` flag that can be used to choose the AE version. Please refer to `--help` option within the binaries for more information.
